client.go

```diff
@@ -31,13 +31,15 @@ const (
 	// ProductionAuthURL is the production auth url
 	ProductionAuthURL = "https://www.dwolla.com/oauth/v2/authenticate"
 	// ProductionTokenURL is the production token url
+	// Deprecated - use https://api.dwolla.com/token moving forward
 	ProductionTokenURL = "https://accounts.dwolla.com/token"
```

Comment:
We can just set this to "https://api.dwolla.com/token" and remove the deprecation comment. Changing the value isn't an API change here.